resources/rafter/charts/rafter-upload-service/README.md

```diff
@@ -0,0 +1,7 @@
+# Rafter Upload Service
```

Comment:
I would suggest changing the chart name from `charts/rafter-upload-service` to `charts/upload-service`. What follows, please update the README title to `# Upload Service`.

Reply:
Changed